go-controller/pkg/ovn/namespace.go

```diff
@@ -336,6 +336,9 @@ func (oc *DefaultNetworkController) updateNamespace(old, newer *kapi.Namespace)
 				errors = append(errors, fmt.Errorf("failed to get all the pods (%v)", err))
 			}
 			for _, pod := range existingPods {
+				if util.PodCompleted(pod) {
```

Comment:
should we also skip host networked and pods that are not scheduled?

Reply:
only in case of pod completed case, the same IP is assigned to new pod which creates conflicts in the data plane, do you think checking other cases make sense here as well ?

Reply:
yeah we shouldn't add SNAT's for pods that are not scheduled to a node. In fact that would result in an error here and updateNamespace would retry, which we dont want to do. For host networked pods, they obviously wouldn't need an SNAT either. I think it makes sense to add these checks as well.

Reply:
sure. done.